src/main/java/com/powsybl/openloadflow/ac/outerloop/AcloadFlowEngine.java

```diff
@@ -187,9 +187,12 @@ public void close() {
         return createNetworks(network, parameters)
                 .stream()
                 .map(n -> {
-                    try (AcloadFlowEngine engine = new AcloadFlowEngine(n, parameters)) {
-                        return engine.run();
+                    if (n.isValid()) {
+                        try (AcloadFlowEngine engine = new AcloadFlowEngine(n, parameters)) {
+                            return engine.run();
+                        }
                     }
+                    return new AcLoadFlowResult(n, 0, 0, NewtonRaphsonStatus.NO_CALCULATION, 0d);
```

Comment:
`0` for the mismatch is maybe not a good choice. Is it possible to put Double.NaN instead?